API documentation

ArkShocer opened the pull request:
- This PR adds some documentation for the rest api.

ArkShocer committed:
Updated README.md, added API.md

ArkShocer committed:
Update API.md

ArkShocer committed:
Added Character and Master API

enteryournamehere commented on Uchu.Api/API.md:
```diff
@@ -0,0 +1,51 @@
+| `instance/list` | Returns detailed list about all `instances` created. | http://localhost:10000/instance/list | None |
+| `instance/list/complete` | Returns detailed list about all `instances` created successfully. | http://localhost:10000/instance/list/complete | None |
+| `instance/commission` | Takes in the `id` of a `world`, and creates an `instance` with that `id`. | http://localhost:10000/instance/commission?= `id` | `id` |
+| `master/decommission` | Stops the `master server` and all `instance`. | http://localhost:10000/master/decommission?= `stopping%20server.` | `stopping%20server.` |
```
Maybe this could be made a bit clearer to explain that `stopping server` here is just an example, and that this text will be logged to the console before exiting

enteryournamehere commented:
World servers also have some API commands available that may be useful to people: https://github.com/uchuserver/uchu/blob/dev/Uchu.World/Api/WorldCommands.cs

ArkShocer committed:
Added world options

- Added world options
- Changed wording for some requests

ArkShocer committed:
Changed some wording

Jettford reviewed (changes requested):
Not sure why I was requested to review but I guess I will, I would add a bit of explanation on the configuration if you can.

ArkShocer committed:
Updated wording

- Added world/zoneStatus for upcoming merge
- Added some more information about the configuration

ArkShocer committed:
Update API.md

ArkShocer committed:
Update API.md

ArkShocer committed:
Merge branch 'UchuServer:dev' into dev
